components/Navigation/navigation.html

```diff
@@ -0,0 +1,119 @@
+            <li class="nav-list">
+                <a href="https://github.com/jatinmore" target="_blank" class="nav-link"><i class="fab fa-github" id="nav-icons"></i></a>
+            </li>
+
+            <li class="nav-list">
+                <a href="https://twitter.com/JatinMore9" target="_blank" class="nav-link"><i class="fab fa-twitter" id="nav-icons"></i></a>
+            </li>
+        </div>
+
+    </div>
+    <div class="flex-container">
+        <div class="sidebar">
+            <div class="aside">
+                <span class="aside-header">Components</span>
+                <ul class="list">
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Avatar-Badges/avatar.html">Avatar</a>
+                    </li>
+
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Avatar-Badges/badge.html">Badge</a>
+                    </li>
+
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Alert-Button/button.html">Button</a>
+                    </li>
+
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Alert-Button/alert.html">Alert</a>
+                    </li>
+
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Card/card.html">Card</a>
+                    </li>
+
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Image-Input/image.html">Image</a>
+                    </li>
+
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Image-Input/input.html">Input</a>
+                    </li>
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Typography/typography.html">Typography</a>
+                    </li>
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/List/list.html">List</a>
+                    </li>
+
+                    <li class="aside-item">
+                        <a class="aside-link active" href="/components/Navigation/navigation.html">Navigation</a>
+                    </li>
+
+                </ul>
+            </div>
+        </div>
+
+        <div class="content-main">
+
+            <span class="description">
+                <h1>Navigation</h1>
+                 <p class="sub-heading font-size-10">
+                    A navigation bar, at its very core, is a list of links used for navigating through various pages.
+                 </p>            
+            </span>
+
+            <div class="container">
+                <div class="description">
+                    <span class="underline">Standard Nav</span>
+
+                </div>
+                <div class="navbar">
```

Comment:
here , instead of using div for navbar , I suggest you can use `<nav>` tag which makes it semantically correct.
You can use this suggestion even for you component library navbar as well.

Otherwise UI looks clean. Good work.